src/App.vue

```diff
@@ -269,38 +261,25 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return process.env.ABOUT_TEXT
   }
 
-  /** Track changes to alteration specific pieces. */
-  private get hasAlterationChanges (): boolean {
-    return (
-      this.hasBusinessNameChanged ||
-      this.hasBusinessTypeChanged
-    )
-  }
-
   /** Whether user is authenticated. */
   private get isAuthenticated (): boolean {
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
-  /** Safety check to ensure that fee summary component is not loaded until there
-   *  is a valid filing type and entity code.
-   */
-  private get showFeeSummary (): boolean {
-    const defaultFilingData = {
-      filingTypeCode: null as FilingCodes,
-      entityType: null as CorpTypeCd,
-      priority: false,
-      waiveFees: false
-    }
-    return this.isFilingChanged && !this.isSame(this.getFilingData, defaultFilingData)
-  }
-
+  /** The error to display in the fee summary component, if any. */
   private get feeSummaryError (): string {
     // TODO: finish this
```

Comment:
I think Cam will finish this.

Anyway, I have a few TODOs in here that I will get to before calling the ticket done.

Reply:
yes sir, i have this in my current work 👍